istio/datadog_checks/istio/istio.py

```diff
@@ -85,6 +94,9 @@ def create_generic_instances(self, instances):
             if 'galley_endpoint' in instance:
                 process_galley_instance = self._create_process_galley_instance(instance)
                 yield process_galley_instance
+            if 'citadel_endpoint' in instance:
+                process_citadel_instance = self._create_process_citadel_instance(instance)
+                yield process_citadel_instance
```

Comment:
Not necessarily related only to this PR, but using generics/yield seems overkill here, returning a list would be enough.

It's only used once and directly converted to list `list(self.create_generic_instances(instances))`.

Reply:
Fair point.